src/main/java/com/iab/gdpr/ConsentStringParser.java

```diff
@@ -258,12 +253,10 @@ private boolean findVendorIdInRange(int vendorId) {
 	}
 
 	/**
-	 * @return a boolean describing if a vendor has consented to a particular vendor. The lowest vendor ID is 1.
-	 *
-	 *         This method, along with {@link isPurposeAllowed} fully describes the user consent for a particular action
-	 *         by a given vendor.
+	 * {@inheritDoc}
 	 */
-	public boolean isVendorAllowed(int vendorId) {
+	@Override
+	public boolean isVendorConsented(int vendorId) {
```

Comment:
in v1--changed the language from `is*Allowed` to `is*Consented` since "allowed" means something specific in v2